### PR TITLE
add caption to gdebrito slide

### PR DESCRIPTION
**Pull request by gdebrito**
Hello, @MorganeB  please review

related: #6

### CODE REVIEW

File: _posts/0000-01-06-gdebrito.md [hunk 1]
```diff
@@ -6,4 +6,4 @@ title: "Welcome to our slide deck!"
 ![Robotocat](https://octodex.github.com/images/Robotocat.png)
 {: .center}
```
Comment (gdebrito) [inline _posts/0000-01-06-gdebrito.md:7]: 🐶

Comment (arnaudolig) [reply _posts/0000-01-06-gdebrito.md:7]: Great stuff mate

**Review by eib-fabianburtard — APPROVED** [review]
Great 🎆

**Review by tdelsaux — APPROVED** [review]
could be better...

**Review by florin-d — APPROVED** [review]
Ok, kill them all!

**Pull request merged**